Gemfile

```diff
@@ -20,6 +20,9 @@ group :development do
   # Can be configured to work on production as well see: https://github.com/MiniProfiler/rack-mini-profiler/blob/master/README.md
   gem 'rack-mini-profiler', '~> 3.0'
   gem "letter_opener"
+  gem "lookbook", ">= 2.3.9"
+  gem "listen"
```

Comment:
Can you drop a comment here that says these are required for hot reloading of lookbook.